romancal/resample/resample_utils.py

```diff
@@ -401,3 +416,31 @@ def resample_range(data_shape, bbox=None):
         ymax = min(data_shape[0] - 1, int(y2 + 0.5))
 
     return xmin, xmax, ymin, ymax
+
+
+def add_var_sky_array(input_models: ModelLibrary):
+    """
+    Add sky variance array to each input model.
+
+    Parameters
+    ----------
+    input_models : ModelLibrary
+        A library of models or a single model to which the sky variance array will be added.
+
+    Returns
+    -------
+    None
+    """
+    input_models = (
+        input_models
+        if isinstance(input_models, ModelLibrary)
+        else ModelLibrary([input_models])
+    )
```

Comment:
```suggestion
    input_models = (
        input_models
        if isinstance(input_models, ModelLibrary)
        else ModelLibrary([input_models])
    )
```
If the function only accepts a `ModelLibrary` (based on the type annotation) then is this necessary?

Reply:
No, it is not.

Initially, the method was supposed to handle both `ModelLibrary` and single datamodels, but eventually I thought that it would be better to sticking with the former only.